### PR TITLE
Feature/teacher crud

### PR DESCRIPTION
**Pull request by acanewind**

### CODE REVIEW

File: live-teaching-web/src/app/subject/subject.service.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  listSubjectXXXXX(
+    offset: number,
+    max: number,
+    filter?: any,
+    sort?: string,
+    order?: any
+  ) {
+    // return this.pagedRestfulService.listPagedRestful( 'review', offset, max, filter, sort, order)
+    return of(null);
+  }
```
Comment (new-bmk) [inline live-teaching-web/src/app/subject/subject.service.ts:26]: fix this code.

**Review by new-bmk — COMMENTED** [review]
I comment code. You should fix code.

File: live-teaching-web/src/app/subject/subject.service.ts (hunk 1 above)
Comment (acanewind) [reply live-teaching-web/src/app/subject/subject.service.ts:26]: already fix. please check

**Review by new-bmk — APPROVED** [review]
ok, done.

**Pull request merged**